### ISSUE TITLE
Enhancement: Add possibility to chat to multiple LLM at once in a multi split view

### ISSUE THREAD

**Issue by madnight** [danny-avila/LibreChat#1838]
### What features would you like to see added?

I would like to chat with GPT-4, Turbo, Gemini, Bing, and others in parallel. I think a 4-way split should be sufficient to allow chatting with 4 LLMs in parallel.

### More details

It could look similar to this:
![image](https://github.com/danny-avila/LibreChat/assets/10064471/d18a443a-d748-41b5-80cd-1a0f3c4d0a2b)

### Which components are impacted by your request?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jerkstorecaller** [comment]
Other than casual benchmarking, what is the benefit of this? I think this isn't a good fit for LibreChat, I hope the dev doesn't get bogged down in every creative idea the users have. Focus on the core use-case (ChatGPT replacement).

**Comment by danny-avila** [comment]
> Other than casual benchmarking, what is the benefit of this? I think this isn't a good fit for LibreChat, I hope the dev doesn't get bogged down in every creative idea the users have. Focus on the core use-case (ChatGPT replacement).

I agree, not planned for now but something like multiple responses at a time, similar to when ChatGPT, is planned. It's a cool feature but happens randomly, and would be cool to have some control over that.

**Comment by madnight** [comment]
> Other than casual benchmarking, what is the benefit of this? 

Well, you can present a problem and consult multiple LLMs, such as GPT-4, Gemini Ultra, and more. Then, you can select the best solution. I used to operate in this manner in the past, and it is significantly better than receiving just one answer from GPT-4. But I can understand that this feature is out of scope for this project.

**Comment by danny-avila** [comment]
Not out of scope just not a priority at the moment. I would like to see something like this in the future
